contracts/multisig/src/msg.rs

```diff
@@ -51,12 +51,11 @@ pub struct Signer {
     pub address: Addr,
     pub weight: Uint256,
     pub pub_key: PublicKey,
-    pub signature: Option<Signature>,
 }
 
 #[cw_serde]
 pub struct Multisig {
     pub state: MultisigState,
     pub quorum: Uint256,
-    pub signers: Vec<Signer>,
+    pub signers: Vec<(Signer, Option<Signature>)>,
```

Comment:
```suggestion
    pub signers_with_sigs: Vec<(Signer, Option<Signature>)>,
```

Reply:
why are signatures optional?

Reply:
This is the response for the `GetMultisig` query, and the behavior is to return all of the signers, even if they haven't signed a particular message yet. So the signature is present iff they signed.